test/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishTrimmedWindowsFormsAndWPFApps.cs

```diff
@@ -108,6 +108,72 @@ public void It_publishes_windows_Forms_app_with_error_suppressed()
                 .NotHaveStdOutContaining(Strings.@TrimmingWindowsFormsIsNotSupported);
         }
 
+        [WindowsOnlyRequiresMSBuildVersionFact("17.0.0.32901")]
+        public void It_publishes_windows_forms_app_with_no_wpf()
```

Comment:
We should also have coverage of an app that uses both WPF and Windows Forms.  It may be possible to do so in multiple ways (set both properties to true, explicitly have a FrameworkReference to the WindowsDesktop framework).

Reply:
Added a test with both profiles set to true. Also validated targeting net8